dev-libs/libcgroup/libcgroup-0.41-r6.ebuild

```diff
@@ -12,19 +12,29 @@ SRC_URI="mirror://sourceforge/project/libcg/${PN}/v${PV}/${P}.tar.bz2"
 LICENSE="LGPL-2.1"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+
 IUSE="+daemon elibc_musl pam static-libs test +tools"
+REQUIRED_USE="daemon? ( tools )"
 
-RDEPEND="pam? ( virtual/pam )"
+# Use mount cgroup to build directory
+# sandbox restricted to trivial build,
+RESTRICT="test"
 
-DEPEND="
-	${RDEPEND}
+BDEPEND="
 	sys-devel/bison
 	sys-devel/flex
 	elibc_musl? ( sys-libs/fts-standalone )
-	"
-REQUIRED_USE="daemon? ( tools )"
+"
+DEPEND="pam? ( virtual/pam )"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-replace-DECLS.patch
+	"${FILESDIR}"/${P}-replace-INLCUDES.patch
+	"${FILESDIR}"/${P}-reorder-headers.patch
+	"${FILESDIR}"/${P}-remove-umask.patch
+)
 
-DOCS=(README_daemon README README_systemd INSTALL)
```

Comment:
I suppose this stops INSTALL from being installed. Installing it sounds weird but maybe this package had a good reason for it?

Reply:
I read through INSTALL and it felt like a pointless file to add, all it does is concern packaging/building.